Description: Expected failing tests
 Some tests can FAIL on machines without network access,
 or TIMEOUT in some cases.
 Some rare tests fail because there is a long standing bug that prevents
 building the package.
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2012-02-19
--- a/test/simple/simple.status
+++ b/test/simple/simple.status
@@ -1 +1,15 @@
 prefix simple
+
+# only fail when some network resources are not available
+test-http-dns-fail: PASS || FAIL
+test-net-connect-timeout: PASS || FAIL
+test-dgram-multicast-multi-process: PASS || FAIL
+test-dgram-broadcast-multi-process: PASS || FAIL
+
+# sometimes these tests TIMEOUT
+test-dgram-pingpong: PASS || TIMEOUT
+test-stream-pipe-multi: PASS || TIMEOUT
+
+# should use [ $arch == arm ] but $arch could be wrong
+test-buffer: PASS || FAIL
+
